docker-hub/image-management.md

```diff
@@ -43,15 +47,13 @@ The Advanced Image Management dashboard provides a snapshot of your existing ima
 By default, the Advanced Image Management dashboard displays inactive images in the repository. To delete an inactive image:
 
 1. Log in to [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"}.
-2. Click **Repositories** from the main menu and select a repository from the list. This displays a banner indicating the number of inactive images in the repository.
-3. Click **View inactive images**. This displays the Advanced Image Management dashboard.
-4. Select the images that you’d like to delete from the repository.
-5. Click the delete icon located above the list of images.
+2. Click **Repositories** from the main menu and select a repository from the list.
+3. Click **General** or **Tag** and click Go to Advanced Image Management from the main menu.
```

Comment:
same comment around **Tag** being in singular form while the tab has it in plural

Reply:
Thanks for catching this @fdiogo, updated!